Platforms/QemuQ35Pkg/QemuFlashFvbServicesRuntimeDxe/FwBlockService.c

```diff
@@ -1129,7 +1131,10 @@ FvbInitialize (
   //
   InstallVirtualAddressChangeHandler ();
 
-  PcdStatus = PcdSetBoolS (PcdOvmfFlashVariablesEnable, TRUE);
-  ASSERT_RETURN_ERROR (PcdStatus);
+  // MU_CHANGE: Abstract dynamic PCD set to support Standalone MM
+  // PcdStatus = PcdSetBoolS (PcdOvmfFlashVariablesEnable, TRUE);
+  // ASSERT_RETURN_ERROR (PcdStatus);

```

Comment:
For all the commented out code segments can we just remove that code? Or do you think there is value in keeping the old code as reference for what was changed?

Reply:
I think it makes sense now. This was made before the removal of OvmfPkg, I will go through them and see if what can be cleaned up in that regard.

Reply:
Went through all commits another round. The comments should be removed now.